doc/api/cli.md

```diff
@@ -282,22 +282,24 @@ added:
 
 Enable experimental `import.meta.resolve()` support.
 
-### `--experimental-json-modules`
+### `--experimental-loader=module`
 
 <!-- YAML
-added: v12.9.0
+added: v9.0.0
```

Comment:
are we sure this was added in node 9?
```sh
Running node v9.11.2 (npm v5.6.0)
node: bad option: --experimental-loader=module
```

Reply:
Don't know, could be wrong, this is from https://github.com/nodejs/node/pull/22104 and not related to this backport.

Reply:
I've opened https://github.com/nodejs/node/pull/42778 to fix that on `master`.